### PR TITLE
ADEN-12994 Extend Eyeota pixel payload with media tags

### PR DESCRIPTION
**Pull request by jbienkowski-fandom**
## Links
https://fandom.atlassian.net/browse/ADEN-12994

## Description
This PR extends Eyeota integration so it sends Site context media tags as key-vals to the `/pixel` endpoint.

E2E tests: https://github.com/Wikia/ad-tests/pull/422

### CODE REVIEW

**Review by szymon-nawrocki — APPROVED** [review]

File: src/ad-services/eyeota/index.ts [hunk 1]
```diff
@@ -52,8 +52,8 @@ export class Eyeota extends BaseServiceSetup {
 
 		let contextTags = '';
 		if (window.fandomContext?.site?.tags) {
-			const { gnre, pform, pub, tv } = window.fandomContext.site.tags;
-			contextTags = parseContextTags({ gnre, pform, pub, tv });
+			const { gnre, media, pform, pub, tv } = window.fandomContext.site.tags;
```
Comment (kwfandom) [inline src/ad-services/eyeota/index.ts:55]:
```suggestion
			const { gnre, media, pform, pub, theme, tv } = window.fandomContext.site.tags;
```

File: src/ad-services/eyeota/index.ts [hunk 2]
```diff
@@ -52,8 +52,8 @@ export class Eyeota extends BaseServiceSetup {
...
+			contextTags = parseContextTags({ gnre, media, pform, pub, tv });
```
Comment (kwfandom) [inline src/ad-services/eyeota/index.ts:56]:
```suggestion
			contextTags = parseContextTags({ gnre, media, pform, pub, theme, tv });
```

File: spec/ad-services/eyeota/eyeota.spec.ts [hunk 3]
```diff
@@ -103,6 +103,7 @@ describe('Eyeota', () => {
 		};
 		const mockedTags = {
 			gnre: ['1', '2', '3'],
+			media: ['web'],
```
Comment (kwfandom) [inline spec/ad-services/eyeota/eyeota.spec.ts:106]:
```suggestion
			media: ['web'],
			theme: ['elf'],
```

File: spec/ad-services/eyeota/eyeota.spec.ts [hunk 4]
```diff
@@ -117,7 +118,7 @@ describe('Eyeota', () => {
 		delete window.fandomContext;
 
 		expect(src).to.equal(
-			'https://ps.eyeota.net/pixel?pid=r8rcb20&sid=fandom&t=ajs&s0v=undefined&gnre=1&gnre=2&gnre=3&pform=xbox&pub=test',
+			'https://ps.eyeota.net/pixel?pid=r8rcb20&sid=fandom&t=ajs&s0v=undefined&gnre=1&gnre=2&gnre=3&media=web&pform=xbox&pub=test',
```
Comment (kwfandom) [inline spec/ad-services/eyeota/eyeota.spec.ts:121]:
```suggestion
			'https://ps.eyeota.net/pixel?pid=r8rcb20&sid=fandom&t=ajs&s0v=undefined&gnre=1&gnre=2&gnre=3&media=web&theme=elf&pform=xbox&pub=test',
```

**Comment by kwfandom** [comment]
_NEF 4/3: please theme key-val as well 🙈. All major Fandom content key-vals are in use by Eyeota mapping, turns out. Thank you!_

File: spec/ad-services/eyeota/eyeota.spec.ts [hunk 5]
```diff
@@ -117,7 +119,7 @@ describe('Eyeota', () => {
 		delete window.fandomContext;
 
 		expect(src).to.equal(
-			'https://ps.eyeota.net/pixel?pid=r8rcb20&sid=fandom&t=ajs&s0v=undefined&gnre=1&gnre=2&gnre=3&pform=xbox&pub=test',
+			'https://ps.eyeota.net/pixel?pid=r8rcb20&sid=fandom&t=ajs&s0v=undefined&gnre=1&gnre=2&gnre=3&media=web&theme=elf&pform=xbox&pub=test',
```
Comment (jbienkowski-fandom) [inline spec/ad-services/eyeota/eyeota.spec.ts:122]:
```suggestion
			'https://ps.eyeota.net/pixel?pid=r8rcb20&sid=fandom&t=ajs&s0v=undefined&gnre=1&gnre=2&gnre=3&media=web&pform=xbox&pub=test&theme=elf',
```

**Review by kwfandom — APPROVED** [review]

**Pull request merged**